README.md

```diff
@@ -1,4 +1,9 @@
-# rsk-transaction-helper
+<img src="./rootstock_logo.png" alt="Rootstock" />
+
+# rootstock-transaction-helper
+![Github CI/CD](https://github.com/rsksmart/rootstock-transaction-helper/actions/workflows/workflow.yml/badge.svg)
+[![CodeQL](https://github.com/rsksmart/rootstock-transaction-helper/workflows/CodeQL/badge.svg)](https://github.com/rsksmart/rootstock-transaction-helper/actions?query=workflow%3ACodeQL)
+
 Utility library to send transactions on RSK using Web3 and Ethereum-js 
```

Comment:
Should this be `Utility library to send transactions on Rootstock using Web3 and Ethereum-js`?